lib/client.js

```diff
@@ -444,6 +542,84 @@ MatrixClient.prototype.listDeviceKeys = function(userId) {
 };
 
 /**
+ * Mark the given device as verified
+ *
+ * @param {string} userId owner of the device
+ * @param {string} deviceId unique identifier for the device
+ */
+MatrixClient.prototype.setDeviceVerified = function(userId, deviceId) {
+    if (!this.sessionStore) {
+        throw new Error("End-to-End encryption disabled");
+    }
+
+    var devices = this.sessionStore.getEndToEndDevicesForUser(userId);
+    if (!devices || !devices[deviceId]) {
+        throw new Error("Unknown device " + userId + ":" + deviceId);
+    }
+
+    var dev = devices[deviceId];
+    if (dev.verified) {
+        return;
+    }
+    dev.verified = true;
+    this.sessionStore.storeEndToEndDevicesForUser(userId, devices);
+};
+
+
+/**
+ * Check if the sender of an event is verified
+ *
+ * @param {MatrixEvent} event event to be checked
+ * @return {boolean} true if the sender of this event has been verified
```

Comment:
You could say, "if the device has been marked as verified using setDeviceVerified", just to make it clear the two APIs are linked?